packages/components/Organisms/Accordion/Accordion.vue

```diff
@@ -4,24 +4,33 @@
     <div class="rpl-accordion__collapse">
       <button class="rpl-accordion__collapse-btn" @click="closeOpenAll">{{ closeOpenLabel }}</button>
     </div>
-    <component :is="isNumbered ? 'ol' : 'ul'" class="rpl-accordion__list">
```

Comment:
Arghh I forgot this one too, can we keep the change from `ol` and `ul` -> `component` as well? It's just a huge chunk of repeated code

Reply:
Hey @waitingallday, I've swapped that back to the single non-repeated list now. Cheers